data/json/items/comestibles.json

```diff
@@ -1684,9 +1684,180 @@
+    "volume": 1,
+    "use_action": {
+      "target": "bone_marrow",
+      "msg": "You snap the bone into two pieces and extract the marrow.",
+      "target_charges": 2,
+      "menu_text": "Extract marrow",
+      "type": "transform"
+    }
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "bone_marrow",
+    "name": "bone marrow",
+    "weight": 55,
+    "color": "white",
+    "spoils_in": 120,
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 2,
+    "nutrition": 15,
+    "description": "A soft, spongy substance found in bones.  Chock-full of good stuff, but it looks and tastes unpleasant.",
+    "price": 0,
+    "charges": 2,
+    "material": "bone",
+    "volume": 1,
+    "fun": -3,
+    "vitamins": [ [ "calcium", 5 ], [ "iron", 10 ] ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "cooked_marrow",
+    "name": "roasted marrow",
+    "weight": 55,
+    "color": "brown",
+    "spoils_in": 168,
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 2,
+    "nutrition": 15,
+    "description": "Marrow from animal bones that's been cooked and prepared.  Tasty!",
+    "price": 0,
+    "material": "bone",
+    "volume": 1,
+    "fun": 5,
+    "vitamins": [ [ "calcium", 5 ], [ "iron", 10 ] ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "bone_tainted",
+    "category": "other",
+    "name": "tainted bone",
+    "weight": 212,
+    "color": "light_gray",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": -1,
+    "nutrition": 2,
+    "description": "A rotten and brittle bone from some unnatural creature or other.",
+    "price": 0,
+    "//": "Not for use in edible recipes, and should require ~200% as much as normal for applicable inedible recipes except for charcoal.",
+    "material": "bone",
+    "volume": 1,
+    "fun": -10,
+    "use_action": {
+      "target": "tainted_marrow",
+      "msg": "You snap the bone into two pieces and extract the marrow.",
+      "target_charges": 2,
+      "menu_text": "Extract marrow",
+      "type": "transform"
+    }
+  },
+  {
+    "type": "GENERIC",
+    "id": "bone_tainted",
+    "category": "other",
+    "name": "tainted bone",
+    "weight": 212,
+    "color": "light_gray",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": -1,
+    "nutrition": 2,
+    "description": "A rotten and brittle bone from some unnatural creature or other.",
+    "price": 0,
+    "//": "Not for use in edible recipes, and should require ~200% as much as normal for applicable inedible recipes except for charcoal.",
+    "material": "bone",
+    "volume": 1,
+    "fun": -10,
+    "use_action": {
+      "target": "tainted_marrow",
+      "msg": "You snap the bone into two pieces and extract the marrow.",
+      "target_charges": 1,
+      "menu_text": "Extract marrow",
+      "type": "transform"
+    }
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "tainted_marrow",
+    "name": "tainted marrow",
+    "weight": 20,
+    "color": "white",
+    "spoils_in": 60,
```

Comment:
I have no idea; I've never eaten bone marrow.

Reply:
You should make some research then :)

Reply:
1. Can you share your source?
2. Was it refrigerated or not? Because of in-game temperature-based rot calculation taking place, all in-game spoil rates **must** be set in ambient room temperature. 
3. This source: https://www.rawpaleodietforum.com/general-discussion/shelf-life-of-bone-marrow/ tells that a week is achievable **while refrigerated**. Unfortunately I have found no more sources, as apparently bone marrow is rarely eaten, and even if - its eaten fresh to retain its taste.

Reply:
That was my slip-up. I probably saw the "refrigerated"part. That's my mistake.